sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -583,13 +621,15 @@ We did not utilize this extension for this matrix API version because sub-group
 - Ronan Keryell: "It would be interesting to investigate whether providing also member functions would simplify the API. Provide both so it is possible to use the best one for each use case, while waiting for https://en.wikipedia.org/wiki/Uniform_Function_Call_Syntax to land into C++?"
 
 - In the future looking APIs, `get_wi_slice` (that is currently under design) returns an owned object. Should this return a view object to make sure the original matrix C is changed after its slices are modified.
-
+- multi_ptr can be constructed from T* since https://github.com/intel/llvm/pull/1183.  However currently this cannot be used with USM for all cases.
+  It is expected that eventually the `joint_matrix_load` and `joint_matrix_store` interfaces will be fully compatible with USM.
```

Comment:
They are already compatible with USM for AMX and DPAS.
Can you specify that this is only the case for tensorcores use case?
Can you also elaborate on the issue for tensorcores?